csaf_2.1/prose/edit/src/schema-elements-01-defs-03-full-product-name.md

```diff
@@ -51,27 +51,27 @@ and `x_generic_uris`, one is mandatory.
         "cpe": {
           // ...
         },
-        "hashes": {
+        "hashes": [
           // ...
-        },
-        "model_numbers": {
+        ],
+        "model_numbers": [
           // ...
-        },
-        "purl": {
+        ],
+        "purls": [
           // ...
-        },
-        "sbom_urls": {
+        ],
+        "sbom_urls": [
           // ...
-        },
-        "serial_numbers": {
+        ],
+        "serial_numbers": [
           // ...
-        },
-        "skus": {
+        ],
+        "skus": [
           // ...
-        },
-        "x_generic_uris": {
+        ],
+        "x_generic_uris": [
           // ...
-        }
```

Comment:
We should do that in a separate commit to keep the feature changes separate from the editorial ones.

Reply:
@mprpic Sorry about the confusion. After looking into that and #787 again, we noticed that the original brackets were correct: The excerpts show the JSON schema, not an instance (CSAF document).

Reply:
Ah, ok, I'll remove that change then.